src/client/components/Registration.jsx

```diff
@@ -0,0 +1,28 @@
+const Registration = () => {
+
+    return( 
+        <main className="registrationPage">
+            <div id="registrationFormDiv">
+                <div id="registrationSell">
+                    <h2>Create your account</h2>
+                    <ul>
+                        <li>Rate your favourite Pokémon</li>
+                        <li>Discuss your ratings</li>
+                        <li>Pick your dream team</li>
+                    </ul>
+                </div>
+                <form id="registrationForm">
+                    <label htmlFor="emailR" className="emailLabelR">Email</label>
+                    <input type="text" name="emailR" id="emailR" className="regInput"></input>
+                    <label htmlFor="usernameR" className="usernameLabelR">Username</label>
+                    <input type="text" name="usernameR" id="usernameR" className="regInput"></input>
```

Comment:
The "R" after username and password in the "name" attribute might cause a work around issue when making requests to the server. Could this just be renamed to username and password?